miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/exception/ValidationException.java

```diff
@@ -0,0 +1,35 @@
+package uk.ac.bbsrc.tgac.miso.service.exception;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ValidationException extends RuntimeException {
+
+  private static final long serialVersionUID = 1L;
+  
+  private final List<ValidationError> errors;
+
+  /**
+   * Constructs a ValidationException to provide one or more errors explaining why validation failed
+   * 
+   * @param errors all of the errors related to the action in question
+   */
+  public ValidationException(List<ValidationError> errors) {
+    super("Validation failed");
+    this.errors = errors;
+  }
+
+  /**
+   * @return a list of the errors explaining why the action in question has failed validation
+   */
+  public List<ValidationError> getErrors() {
+    return errors;
+  }
+
+  public Map<String, String> getErrorsByField() {
+    return errors.stream()
+        .collect(Collectors.toMap(err -> err.getProperty(), err -> err.getMessage(), (msg1, msg2) -> msg1 + "\n" + msg2));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)